samples/notebooks/powershell/Samples/Using-Mocrosoft-DataFrame.dib

```diff
@@ -0,0 +1,77 @@
+#!meta
```

Comment:
You have a typo in the file name: `Mocrosoft`

You might also consider making this a `.ipynb` and saving it with the outputs. GitHub will display this nicely and people can understand the sample without having to run it.

Reply:
I think I fixed it